modules/op-release-notes-1-7.adoc

```diff
@@ -0,0 +1,307 @@
+
+* Pull request status. When iterating over a pull request, the status and control is executed on the platform hosting the Git repository.
+
+* GitHub Checks API to set the status of a pipeline run, including rechecks.
+
+* GitHub Pull Request and Commit events. 
+
+* Pull request actions in comments, such as `/retest`.
+
+* Git events filtering, and a separate pipeline for each event.
+
+* Automatic task resolution in {pipelines-shortname} (local tasks, Tekton Hub, and remote URLs).
+
+* Use of GitHub blobs and objects API for retrieving configurations.
+
+* Access Control List (ACL) over a GitHub organization, or using a Prow style `OWNER` file.
+
+* The `tkn-pac` plugin for the `tkn` CLI tool, which you can use to manage {pac} repositories and bootstrapping.
+
+* Support for GitHub Application, GitHub Webhook, Bitbucket Server, and Bitbucket Cloud.
+
+[id="deprecated-features-1-7_{context}"]
+== Deprecated features
+
+// Pipelines
+* Breaking change: The `disable-working-directory-overwrite` and `disable-home-env-overwrite` fields in the `TektonConfig` custom resource (CR) is removed. As a result, the `$HOME` environment variable and `workingDir` parameter are unavailable.
+// https://github.com/tektoncd/pipeline/pull/4587
+
+* The `Conditions` custom resource definition (CRD) type will be removed in a future release. Instead, use the recommended `When` expression.
+// issue # unknown; discussed in Slack.
+
+// Triggers
+* Breaking change: Triggers validates and gives error if you provide empty `EventListener` and `TriggerBinding` specifications.
+// https://github.com/tektoncd/triggers/pull/1277 and https://github.com/tektoncd/triggers/pull/1264
+
+
+[id="known-issues-1-7_{context}"]
+== Known issues
+
+// Pipelines
+* Implicit parameter mapping incorrectly passes parameters from the top-level `Pipeline` or `PipelineRun` definitions to the `taskRef` tasks. Mapping should only occur from a top-level resource to tasks with in-line `taskSpec` specifications. This issue only affects users who have set `enable-api-fields` feature flag to `alpha`.
+// https://github.com/tektoncd/pipeline/issues/4483
+
+
+
+[id="fixed-issues-1-7_{context}"]
+== Fixed issues
+
+// Pipelines
+* With this fix, metadata such as `labels` and `annotations` from task and pipeline definitions does not propagate to task runs and pipeline runs.
+// https://github.com/tektoncd/pipeline/pull/4478
+
+* With this fix, if the `timeouts.tasks` field or the `timeouts.finally` field is set to `0`, then the `timeouts.pipeline` is also set to `0`. 
+// https://github.com/tektoncd/pipeline/pull/4539
+
+* With this fix, the `-x` set flag is removed from scripts that do not use a shebang. The fix reduces potential data leak from script execution.
+// https://github.com/tektoncd/pipeline/pull/4451
+
+* With this fix, any backslash character present in the usernames in Git credentials will be escaped with an additional backslash in the `.gitconfig` file.
```

Comment:
```suggestion
* With this fix, any backslash character present in the usernames in Git credentials is escaped with an additional backslash in the `.gitconfig` file.
```
Are you saying the system does this, or that the user can do it by adding a backslash?

Reply:
System.